### PR TITLE
Modify script to avoid deploy error

### PR DESCRIPTION
**Pull request by sandybemonkey**
Adding a response 200 on "/" path to avoid deploy faillure.

### CODE REVIEW

File: nodejs_ES6/src/app.js [hunk 1]
```diff
@@ -20,9 +20,12 @@ if (process.env.NODE_ENV !== 'test') {
 }
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use('/', checkAccessToken);
+app.use('/dgfip', checkAccessToken);
```
Comment (rdubigny) [inline nodejs_ES6/src/app.js]: this instruction seems to mean "we use checkAccessToken for dgfip routes only" but what we want is "we use checkAccessToken everywhere except for the ping route".

File: nodejs_ES6/src/app.js [hunk 2]
```diff
@@ -20,9 +20,12 @@ if (process.env.NODE_ENV !== 'test') {
...
 
 // Setup routing (see https://expressjs.com/en/guide/routing.html)
+app.get('/', (req, res) => {
+  res.sendStatus(200)
```
Comment (rdubigny) [inline nodejs_ES6/src/app.js]: 2 lint errors : plz configure your IDE

Comment (rdubigny) [inline nodejs_ES6/src/app.js]: I suggest you add a comment on the purpose of this route

File: nodejs_ES6/src/app.js (hunk 1 above)
Comment (sandybemonkey) [reply nodejs_ES6/src/app.js]: We only have two routes so i think it's good to do this . but we could do something like app.all('*',checkAccessToken) and check in checkAccessToken if req.path = to "/" we use next()

Comment (sandybemonkey) [reply nodejs_ES6/src/app.js]: Going back to the old app.use('/', checkAccessToken); will end with a deploy's error.

**Pull request closed without merging**